canonical-websites/maas.io
# Update /tour page

## Done

Update /tour as per copy doc

## QA

./run and go to http://0.0.0.0:8006/tour
Check page with copy doc

NB: awaiting some images to replace placeholders

## Issue / Card

Copy doc: https://docs.google.com/document/d/1KcYwNK0pJyiDVEDq4vTpv6BX_rpSDnxUxtAvAnsRKyw/edit#
Issue: https://github.com/CanonicalLtd/MAAS-design/issues/189
